operations/maintainer.go

```diff
@@ -515,3 +543,16 @@ func (om *Maintainer) markStuckOperationsFailed() {
 
 	log.C(om.smCtx).Debug("Finished marking stuck operations as failed")
 }
+
+func getObjectListIDs(lastOpsList types.ObjectList) []string {
```

Comment:
this is not specific to last ops, and is already implemented in base_controller, maybe it would be better to extract to some common util? if not, is the nil check needed? and why notInIds?